api/tests/Api/CampCollaborations/CreateCampCollaborationTest.php

```diff
@@ -369,6 +369,52 @@ public function testCreateCampCollaborationValidatesMissingUserAndInviteEmail()
         ]);
     }
 
+    public function testCreateCampCollaborationValidatesMissingUserAndInviteEmailAndIncludesTranslationInfo() {
+        static::createClientWithCredentials()->request('POST', '/camp_collaborations', ['json' => $this->getExampleWritePayload([], ['user'])]);
+
+        $this->assertResponseStatusCodeSame(422);
+        $this->assertJsonContains([
+            'violations' => [
+                [
+                    'i18n' => [
+                        'key' => 'app.validator.asserteitherisnull',
+                        'parameters' => [
+                            '{{ other }}' => 'user',
+                        ],
+                        'translations' => [
+                            'en' => 'Either this value or user should not be null.',
+                            'en_CH_scout' => 'Either this value or user should not be null.',
+                            'de' => 'Either this value or user should not be null.',
+                            'de_CH_scout' => 'Either this value or user should not be null.',
+                            'fr' => 'Either this value or user should not be null.',
+                            'fr_CH_scout' => 'Either this value or user should not be null.',
+                            'it' => 'Either this value or user should not be null.',
+                            'it_CH_scout' => 'Either this value or user should not be null.',
```

Comment:
Alternative: Only list the ones that have an actual translation and let the client decide on the fallback? Now it's a bit difficult for a client to know if a specific translation contains a real translation or only a fallback.

Reply:
done